composer/tools/copy_environment.py

```diff
@@ -30,20 +30,28 @@
 import argparse
 import ast
 import base64
+import ConfigParser
+import contextlib
 import json
+import os
 import re
 import shutil
+import StringIO
```

Comment:
`import six` and use [six.StringIO](https://pythonhosted.org/six/#six.StringIO) in the script.